national-resources/no_resources.md

```diff
@@ -0,0 +1,337 @@
+  - name:  National Graduate School in Infection Biology and Antimicrobials (IBA)
+    registry: Other
+    url: https://www.ibaschool.no/
+  - name: Turning the Tide of Antimicrobial Resistance (TTA) courses
+    registry: Other
+    url: https://www.ttanetwork.net/events/courses/
+  - name: Medical microbiology and infectious diseases course at Kristiania University College
+    registry: Other
+    url: https://www.kristiania.no/en/syllabus/kristiania-professional-college-/tertiary-vocational-education/ysh-02/medical-microbiology-and-infectious-diseases/
+  - name: Training for the infectious diseases speciality in Norway (Open Access article, Solberg 2000)
+    registry: Other
+    url: https://www.clinicalmicrobiologyandinfection.com/article/S1198-743X(14)63185-2/fulltext
+  - name: Introduction to infectious disease modelling course at University of Oslo (UiO)
+    registry: Other
+    url: https://www.uio.no/studier/emner/medisin/med/MF9120/
+  - name: Learning resources by infeksjonskontroll.no
+    registry: Other
+    url: https://www.infeksjonskontroll.no/kompetanse
+
+# Refer to entries of the "main_tool_ and_resource_table" if institutions, organizations and projects from the country contribute to the development of international tools and resources. 
+
+national_resources: 
+  - name: SARS-CoV-2 Database
+    description: Norwegian SARS-CoV-2 database
+    how_to_access: Open for everyone
+    instance_of: <!--- REPLACE THIS with the tool name of which this resource is an instance of, taken from the all tools and resources page --->
+    related_pages:
+        # More information on which page_id you can use can be found at https://www.infectious-diseases-toolkit.org/contribute/website_overview 
+        human_biomolecular_data: [<!---REPLACE THIS with the page IDs of the human_biomolecular_data pages that you want to list here as related pages--->]
+        human_clinical_and_health_data: [<!---REPLACE THIS with the page IDs of the human_clinical_and_health_data pages that you want to list here as related pages--->]
+        socioeconomic_data: [<!---REPLACE THIS with the page IDs of the socioeconomic_data pages that you want to list here as related pages--->]
+        pathogen_characterisation: [<!---REPLACE THIS with the page IDs of the pathogen_characterisation pages that you want to list here as related pages--->]
+        showcase: [<!---REPLACE THIS with the page IDs of the showcase pages that you want to list here as related pages--->]
+    url: https://covid19.sfb.uit.no/
+    registry:
+      fairsharing: 10.25504/FAIRsharing.668458
+
+  - name: Folkehelseinstituttet (FHI)
+    description: Norwegian Institute of Public Health (NIPH) portal for infectious disease information
+    how_to_access: Open Access
+    instance_of: <!--- REPLACE THIS with the tool name of which this resource is an instance of, taken from the all tools and resources page --->
```

Comment:
```suggestion
    instance_of: 
```

In general, the "REPLACE THIS"  should be replaced, either with nothing, or with something :)